test/unit/modules/sfc/sfc-parser.spec.js

```diff
@@ -87,7 +90,7 @@ describe('Single File Component parser', () => {
         <my-button ref="button">Hello</my-button>
       </example>
       <example name="with props">
-        <my-button color="red">Hello</my-button>
+        <template lang="en" color="red">Hello</template>
```

Comment:
I suggest to add this into another test, something like `doesn't parse template inside custom blocks`